src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -0,0 +1,499 @@
+---
+title: Configure control plane monitoring
+tags:
+  - Integrations
+  - Kubernetes integration
+  - Installation
+metaDescription: How to configure control plane monitoring for your Kubernetes integration with New Relic.
+redirects:
+  - /docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring
+---
+
+[New Relic](/docs/infrastructure/new-relic-infrastructure/getting-started/introduction-new-relic-infrastructure) provides [control plane](http://kubernetes.io/docs/concepts/#kubernetes-control-plane) support for your Kubernetes integration, allowing you to monitor and collect metrics from your cluster's control plane components. That data can then be found in New Relic and used to [create queries and charts](/docs/using-new-relic/data/understand-data/query-new-relic-data).
+
+<Callout variant="tip">
+  Unless otherwise specified, this page refers to the [Kubernetes integration v3](/docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3). Details on how to configure control plane monitoring for v2 can be found in a specific section below.
+</Callout>
+
+## Features [#control-plane-integration-features]
+
+We monitor and collect [metrics](http://docs.newrelic.com/docs/integrations/kubernetes-integration/understand-use-data/understand-use-data#metrics) from the following control plane components:
+
+* **ETCD**: leader information, resident memory size, number of OS threads, consensus proposals data, etc. For a list of supported metrics, see [ETCD data](/docs/integrations/kubernetes-integration/understand-use-data/understand-use-data#etcd-data).
+* **API server**: rate of `apiserver` requests, breakdown of `apiserver` requests by HTTP method and response code, etc. For the complete list of supported metrics, see [API server data](/docs/integrations/kubernetes-integration/understand-use-data/understand-use-data#api-server-data).
+* **Scheduler**: requested CPU/memory vs available on the node, tolerations to taints, any set affinity or anti-affinity, etc. For the complete list of supported metrics, see [Scheduler data](/docs/integrations/kubernetes-integration/understand-use-data/understand-use-data#scheduler-data).
+* **Controller manager**: resident memory size, number of OS threads created, goroutines currently existing, etc. For the complete list of supported metrics, see [Controller manager data](/docs/integrations/kubernetes-integration/understand-use-data/understand-use-data#controller-manager-data).
+
+## Compatibility and requirements [#compatibility]
+
+* Control plane monitoring requires [Kubernetes integration version 1.16.0 or higher](/docs/release-notes/infrastructure-release-notes/kubernetes-integration-release-notes/).
```

Comment:
does it make sense to keep this info ([Kubernetes integration version 1.16.0 or higher]) here?  There is a compatibility and requirements section already